content/docs/command-reference/commit.md

```diff
@@ -36,23 +36,25 @@ detailed below.
 - It's always possible to manually execute the source code used in a stage
   without DVC (outputs should be unprotected or removed first in certain cases,
   see `dvc unprotect`). Once a desirable result is reached, use `dvc add` or
-  `dvc commit` as appropriate to update DVC-files and store changed data to the
-  cache.
+  `dvc commit` as appropriate to update the `dvc.lock` or `.dvc` files and store
+  changed data to the cache.
 
 - Sometimes we want to edit source code, config, or data files in a way that
   doesn't cause changes in the results of their data pipeline. We might write
   add code comments, change indentation, remove some debugging printouts, or any
   other change that doesn't cause changed stage outputs. However, DVC will
-  notice that some <abbr>dependencies</abbr> and have changed, and expect you to
+  notice that some <abbr>dependencies</abbr> have changed, and expect you to
   reproduce the whole pipeline. If you're sure no pipeline results would change,
-  just use `dvc commit` to force update the related DVC-files and cache.
+  just use `dvc commit` to force update the `dvc.lock` or the related `.dvc`
+  file and cache.
 
 Let's take a look at what is happening in the first scenario closely. Normally
 DVC commands like `dvc add`, `dvc repro` or `dvc run` commit the data to the
-<abbr>cache</abbr> after creating a DVC-file. What _commit_ means is that DVC:
+<abbr>cache</abbr> after creating or updating the `dvc.lock` or `.dvc` file.
```

Comment:
the->a  like it was before, please.

> Different articles mean different things. The commands mentioned above don't work with one specific "THE" dvc.lock/etc file. They each work with A (generic) xyz file. See https://owl.purdue.edu/owl/general_writing/grammar/using_articles.html

Reply:
Yeah the context of line it talks about general a file so 'the' can't be used. Thanks.

Reply:
Yep. I'm not even a grammar expert or even a professional English writer, formally speaking. But I feel like these basic grammar details, even when small, can really jump and distract the reader.